scripts/cmsregister.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Scripts should always use `sh`. I don't run bash or Linux on my main development machines and we deploy on `OpenBSD`.